app/views/compute_resources/show/_ovirt.html.erb

```diff
@@ -2,3 +2,7 @@
   <td><%= _("URL") %></td>
   <td><%= @compute_resource.url %></td>
 </tr>
+<tr>
+  <td><%= _("Operating systems API supported?") %></td>
+  <td><%= @compute_resource.supports_operating_systems? ? _("Yes") : _("No") %></td>
```

Comment:
I went with the green/red icon: good idea - we don't need to mess with translations